src/codeview.jl

```diff
@@ -148,7 +147,7 @@ function cthulhu_typed(io::IO, debuginfo::Symbol,
         maxcost = Core.Compiler.statement_costs!(cst, code, src, Any[mi.sparam_vals...], false, params)
         nd = ndigits(maxcost)
         _lineprinter = lineprinter(src)
-        function preprinter(io, linestart, idx)
+        preprinter = function (io, linestart, idx)
```

Comment:
Was there a reason for this change? I generally prefer the former since it will give more informative backtraces

Reply:
That should only effect the case where we actually define different methods, but I see your point

Reply:
ah, I see. Then let's try to keep more informative backtraces.